guides/ci.md

```diff
@@ -0,0 +1,70 @@
+# Continuous Integration and automated checks
+
+This guide explains what runs locally versus in CI, how visual testing works, and how to act on failures.
+
+## Overview
+
+- Local: `yarn test` (or `npm run test`) via `dotrun test` runs spelling, formatting/linting, Parker metrics, and Jest tests. Some tasks (e.g., Percy) are CI-only.
+- CI (GitHub Actions): Percy visual tests run on PRs that meet selectivity rules or when labeled. See [Percy workflow](/guides/percy-workflow.md).
+- Legacy deployment jobs run in Jenkins for PS5 (high-level only; see [Deployment](/guides/deployment.md)).
+
+## Local checks
+
+Typical commands:
+
+```
+dotrun test
+yarn test
+```
+
+Locally you’ll see failures from:
+
+- Spelling (`mdspell`): currently configured with en‑GB; our policy for new docs is en‑US. Use `.spelling` to ignore valid en‑US words and technical terms. See [Language policy](/guides/language.md).
+- Format/lint (e.g., Prettier/Stylelint): fix and re-run.
+- Parker CSS metrics: see below.
+- Jest tests: snapshot configuration tests; see below.
+
+## Visual testing (Percy)
+
+Percy runs visual snapshot tests against the examples site. On PRs, it triggers when:
+
+- The PR targets `main`, and either:
+  - It passes the selectivity filters (changes to `scss/`, `templates/docs/examples/`, `templates/_macros/`, `tokens/`, or `sd.config.json`, and it’s not a draft), or
+  - The PR carries the label `Review: Percy needed`.
+
+Key points:
+
+- Examples are discovered by `snapshots.js` and include both regular and `standalone/` variants for many paths.
+- Combined examples (`combined.html`) reduce snapshot counts by aggregating variants; this helps control quota.
+- If any examples in a group are “responsive,” Percy will capture additional widths for the default theme. See [Percy workflow](/guides/percy-workflow.md) for device widths, themes, and combined-behavior rules.
```

Comment:
Define what it means for an example to be responsive (has medium breakpoint-specific styling).